lib/mutant/ast/meta/send.rb

```diff
@@ -29,6 +30,13 @@ def proc?
           naked_proc? || proc_new?
         end
 
+        # Test if message receiver is :T
+        #
+        # @return [Boolean]
+        def sorbet_receiver?
+          SORBET_RECEIVERS.include? receiver
```

Comment:
Explicit parens please in method bodies, in-line with this entire lib.

Reply:
Sure!